packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -329,6 +329,9 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
         browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-field');
```

Comment:
This is personal preference, but I notice date picker specs use `browser.refresh` to reset state. Because these tests manipulate the same test component, these repeated `browser.url` calls are unnecessary - we can update the repeated `browser.url` calls to `browser.refresh()`, which eliminates the after blocks. This removes extra code and stays consistent with the other tests.
```suggestion
         browser.refresh()
```

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/962/commits/018cda40ca19f0ca53ec7faff8475c064edc0930

Reply:
sorry if it wasn't clear, but if we use refresh we can remove the after blocks

Reply:
Further Fixed here https://github.com/cerner/terra-framework/pull/962/commits/0e36b9e40797140ba28864a7d6c87d38f776f3be

Reply:
Unfurther Fixed here https://github.com/cerner/terra-framework/pull/962/commits/78ec9821e8de85d34678c4514e45c0b21db37209

as per discussion here https://github.com/cerner/terra-framework/pull/962#discussion_r342785207